src/engine/bank02.asm

```diff
@@ -1,12 +1,12 @@
 Func_8000: ; 8000 (2:4000)
-.asm_8000
+.begin
```

Comment:
I think you can remove this local label, and make its references to its functional label. But I guess,  for its logical clarity, it could be better to leave.

Reply:
That's a good point, it does seem redundant.